vagrant/provision-minion.sh

```diff
@@ -44,6 +45,8 @@ cp -r /vagrant/openshift.local.certificates /
 chown -R vagrant.vagrant /openshift.local.certificates
 sed -ie "s/10.0.2.15/${MASTER_IP}/g" /openshift.local.certificates/admin/.kubeconfig
```

Comment:
I would like to handle the cleanup separately, since the current issue being addressed in this PR is a blocker. However, if you are sure about this line no longer being required, I can remove it in this PR.